sync-core/src/main/java/com/cloudant/sync/sqlite/SQLDatabaseQueue.java

```diff
@@ -61,6 +66,22 @@ public Object call() throws Exception {
 
     }
 
+    public int getVersion() throws SQLException {
```

Comment:
javadoc? Something like `Returns the schema version of the underlying SQL database`